src/main/java/de/tum/in/www1/artemis/domain/Feedback.java

```diff
@@ -207,6 +215,22 @@ public void setGradingInstruction(GradingInstruction gradingInstruction) {
         this.gradingInstruction = gradingInstruction;
     }
 
+    public List<TextAssessmentConflict> getFirst_conflicts() {
+        return first_conflicts;
+    }
+
+    public void setFirst_conflicts(List<TextAssessmentConflict> first_conflicts) {
+        this.first_conflicts = first_conflicts;
+    }
+
+    public List<TextAssessmentConflict> getSecond_conflicts() {
```

Comment:
Codacy found an issue: [The instance method name 'getSecond_conflicts' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968856)